core/src/main/resources/db/migration/V1__initial_schema.sql

```diff
@@ -15,6 +15,8 @@ CREATE TABLE Package (
     description text,
     -- Free form information about the vendor
     vendor text,
+    -- Cryptographic signature
+    signature text,
```

Comment:
My understanding of migrations is that they are used to 'upgrade' a database which has live data in it. Since SOTA is still under development, I decided changing the initial schema was more appropriate. However, I'm happy to add a migration if you prefer.